src/RA_Core.cpp

```diff
@@ -268,7 +268,7 @@ static unsigned int IdentifyRom(const BYTE* pROM, unsigned int nROMSize, std::st
     }
     else
     {
-        sCurrentROMMD5 = RAGenerateMD5(pROM, nROMSize);
+        sCurrentROMMD5 = RAGenerateMD5(gsl::make_span(pROM, nROMSize));
```

Comment:
I was trying to learn about spans.
It's like an array but doesn't decay. There will be std::span in the next C++, this is just a placeholder.

This lower params in functions that take pointer and size, instead we can just use span since size is always retained.

Reply:
Will `gsl::make_span` work on unix systems? I'm concerned all this gsl stuff you're adding is actually making it harder to ever have a linux version of the dll. That was one of the primary motivations for switching to an MVVM model.

Reply:
This is taken direclty from Microsoft's GSL repo readme, so yes it works on linux. They've been trying to be cross-platform ever since .NET Core was a thing in 2015.
![image](https://user-images.githubusercontent.com/18649925/51081381-3d52a580-16bc-11e9-979a-f0c851262cc7.png)